tortoisebeacon/handlers.go

```diff
@@ -295,7 +295,8 @@ func (tb *TortoiseBeacon) verifyFirstVotingMessage(message FirstVotingMessage, c
 
 	// TODO(nkryuchkov): Ensure that epoch is the same.
 
-	atxID, err := tb.atxDB.GetNodeAtxIDForEpoch(minerPK.String(), currentEpoch-1)
+	nodeID := types.NodeID{Key: minerPK.String()}
```

Comment:
i dont think that this is a good solution to use nodeID. someone can change the implementation of the GtNodeAtxIDForEpoch to rely on VRFPublicKey and it will be perfectly valid, but introduce failures in tortoisebeacon

Reply:
What solution would you suggest?

Reply:
string was better, still ambiguous in other way but better than nodeID. 
in my opinion the best would be to use []byte, but i don't imply that we should switch to []byte in this change.